setup.py

```diff
@@ -100,8 +100,6 @@ def extras_require():
     packages=find_packages(exclude=["tests.*", "tests"]),
     install_requires=install_requires(),
     extras_require=extras_require(),
-    setup_requires=["pytest-runner"],
-    tests_require=["pytest"],
```

Comment:
these aren't required anymore?

Reply:
No, per the deprecation notice [here](https://pypi.org/project/pytest-runner/), those are now actively discouraged.